From: cgzones <cgzones@googlemail.com>
Date: Thu, 20 Apr 2017 23:58:06 +0200
Subject: fix printf argument

found by cppcheck:
[src/cfg.c:66]: (warning) %d in format string (no. 2) requires 'int' but the argument type is 'unsigned int'.

(cherry picked from commit c6f035e98b5976650c36d633c8f8bd6e076147a3)
---
 src/cfg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/cfg.c b/src/cfg.c
index 0195059..0469e85 100644
--- a/src/cfg.c
+++ b/src/cfg.c
@@ -63,7 +63,7 @@ void printcfgfile(void)
 	printf("#MaxBWeth0 8\n");
 
 	while (p != NULL) {
-		printf("MaxBW%s %d\n", p->interface, p->limit);
+		printf("MaxBW%s %u\n", p->interface, p->limit);
 		p = p->next;
 	}
 
